DEVELOPING.MD

```diff
@@ -46,28 +47,28 @@ Before getting started, it is suggested to create an issue in the Rancher/Ranche
 - [ ] Update chart in rancher/charts
-- [ ] Update chart in rancher/rke2-charts
-- [ ] Updated RKE2 to consume new chart
+- [ ] Update chart in rancher/rke2-charts (RKE2 team)
+- [ ] Updated RKE2 to consume new chart (RKE2 team)
 ```
 
-Then for every PR, link back to this issue so everything can be tracked. When you get to RKE2, you will need to create a separate issue in the RKE2 repository in addition to the Rancher one. 
+For every PR, link back to this issue so everything can be tracked. Whoever does the RKE2 updates will need to create a separate issue in the RKE2 repository in addition to the Rancher one. 
 
 Here are some examples:
 
-* Rancher Issue: https://github.com/rancher/rancher/issues/38188
-* RKE2 issue: https://github.com/rancher/rke2/issues/2867
+* [Rancher Issue](https://github.com/rancher/rancher/issues/38188)
+* [RKE2 issue](https://github.com/rancher/rke2/issues/2867)
 
 ### Step 1: Checking upstream manifests
 
 There are two goals when checking upstream manifests. The first goal is to check to see if any image versions have been updated and the second is to check if any changes in the manifests have occured. 
 
 If you find that image versions have been updated, then you will need to make a pull request to the [image-mirror](https://github.com/rancher/image-mirror) repository first before proceeding. [Here](https://github.com/rancher/image-mirror/pull/261) is an example PR and follow the requirements outlined in the PR template for those changes. After the images have been checked and image-mirror updated with any new images or image versions, then proceed to check check the manifests.
 
-Checking the manifests is best done by cloning down the branch for the version that you want to compare with the helm templates. Then using a diff tool like, Kdiff/Meld/WinMerge, compare each manifest directory with the helm chart directory. Look at the diffs and decide if that diff is brining in a modification or addition that needs to be added to the charts in this repository. If you determine the diff needs brought over, then do so. 
+Checking the manifests is best done by doing a tag compare in each upstream chart. Compare the chart tag, that the rancher vsphere chart is based off of, with the latest tag in upstream. This compares the manifest directories. If there is a change that needs to be added to charts in this repository, bring it in. [Here](https://github.com/kubernetes/cloud-provider-vsphere/compare/v1.25.0...v1.24.2) is an example.
 
-Finally, make sure that the unit tests for each chart has test cases added to existing tests, test are updated, or new tests are added based on the set of changes.
+Finally, make sure that the unit tests for each chart have been updated, or new tests or test cases are added based on the set of changes. Increment the chart version, if necessary.
```

Comment:
How do we know if incrementing the chart version is necessary?

Reply:
AFAIK we need to update the chart version anytime that a new tag of CSI/CPI is released and we update to that version of upstream.

Reply:
Could we add that requirement here?